Tasks/ServiceFabricUpdateManifestsV2/task.json

```diff
@@ -125,6 +125,14 @@
             "visibleRule": "updateType = Manifest versions && updateOnlyChanged = true",
             "helpMarkDown": "Always download a new copy of the artifact. Otherwise use an existing copy, if present."
         },
+        {
+            "name": "imageNamesPath",
+            "type": "filePath",
+            "label": "Image Names Path",
+            "required": false,
+            "visibleRule": "updateType = Docker image settings",
+            "helpMarkDown": "Path to a text file that contains the names of the Docker images associated with the Service Fabric application that should be updated with digests. Each image name must be on its own line and must be in the same order as the digests in Image Digests file."
```

Comment:
Should we also mention that this file is output of sfproj build task. That way it will be easier to grasp.